spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/adapter/MessageListenerAdapter.java

```diff
@@ -208,6 +213,45 @@ protected String getDefaultListenerMethod() {
 		return this.defaultListenerMethod;
 	}
 
+
+	/**
+	 * Set the mapping of queue name or consumer tag to method name. The first lookup
+	 * is by queue name, if that returns null, we lookup by consumer tag, if that
+	 * returns null, the {@link #setDefaultListenerMethod(String) defaultListenerMethod}
+	 * is used.
+	 * @param queueOrTagToMethodName the map.
+	 * @since 1.5
+	 */
+	public void setQueueOrTagToMethodName(Map<String, String> queueOrTagToMethodName) {
+		this.queueOrTagToMethodName.putAll(queueOrTagToMethodName);
+	}
+
+	/**
+	 * Add the mapping of a queue name or consumer tag to a method name. The first lookup
+	 * is by queue name, if that returns null, we lookup by consumer tag, if that
+	 * returns null, the {@link #setDefaultListenerMethod(String) defaultListenerMethod}
+	 * is used.
+	 * @param queueOrTag The queue name or consumer tag.
+	 * @param methodName The method name.
+	 * @since 1.5
+	 */
+	public void addQueueOrTagToMethodName(String queueOrTag, String methodName) {
+		this.queueOrTagToMethodName.put(queueOrTag, methodName);
+	}
+
+	/**
+	 * Add the mapping of a queue name or consumer tag to a method name. The first lookup
```

Comment:
s/Add/Remove and so on.
In other words: the wrong JavaDocs.
Can be fixed on merge if that's all.